### PR TITLE
Consume behov

### PR DESCRIPTION
**Pull request by atleno**

### CODE REVIEW

File: src/main/kotlin/no/nav/dagpenger/regel/api/SerDes.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package no.nav.dagpenger.regel.api
```
Comment (geiralund) [inline src/main/kotlin/no/nav/dagpenger/regel/api/SerDes.kt]: Missing tests for these?

File: src/main/kotlin/no/nav/dagpenger/regel/api/KafkaDagpengerBehovConsumer.kt [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    val env: Environment,
+    val tasks: Tasks,
+    val minsteinntektBeregninger: MinsteinntektBeregninger
+) {
+
+    val SERVICE_APP_ID = "dp-regel-api"
+    private lateinit var streams: KafkaStreams
+    fun start() {
+        streams = KafkaStreams(buildTopology(), this.getConfig())
+        streams.setUncaughtExceptionHandler { t, e -> System.exit(0) }
```
Comment (nicokk) [inline src/main/kotlin/no/nav/dagpenger/regel/api/KafkaDagpengerBehovConsumer.kt]: Should we log something here?

Comment (geiralund) [inline src/main/kotlin/no/nav/dagpenger/regel/api/KafkaDagpengerBehovConsumer.kt]: log error first?

File: src/main/kotlin/no/nav/dagpenger/regel/api/KafkaDagpengerBehovConsumer.kt [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        val builder = StreamsBuilder()
+
+        val stream = builder.stream(
+            Topics.DAGPENGER_BEHOV_EVENT.name,
+            Consumed.with(Serdes.StringSerde(), Serdes.serdeFrom(JsonSerializer(), JsonDeserializer()))
+        )
+
+        stream
+            .peek { key, value -> LOGGER.info("Processing behov with id ${value.behovId}") }
+            //.filter { _, behov -> shouldBeProcessed(behov) }
```
Comment (nicokk) [inline src/main/kotlin/no/nav/dagpenger/regel/api/KafkaDagpengerBehovConsumer.kt]:
```suggestion
            .filter { _, behov -> hasNeededMinsteinntektResult(behov) }
```
, and then remove the check in `storeResult()`?

Comment (geiralund) [reply src/main/kotlin/no/nav/dagpenger/regel/api/KafkaDagpengerBehovConsumer.kt]: +1

**Pull request merged**